src/services/connector/serverless.yml

```diff
@@ -225,11 +233,26 @@ resources:
       Type: "AWS::Logs::LogGroup"
       Properties:
         LogGroupName: /aws/fargate/${self:service}-${sls:stage}-kafka-connect
+    KafkaConnectUiWorkerLogGroup: # created a new log group for  the kafka connect ui
+      Type: "AWS::Logs::LogGroup"
+      Properties:
+        LogGroupName: /aws/fargate/${self:service}-${sls:stage}-kafka-connect-ui
     KafkaConnectWorkerSecurityGroup:
       Type: AWS::EC2::SecurityGroup
       Properties:
         GroupDescription: Security Group for the Fargate Connect Workers.
         VpcId: ${self:custom.vpc.id}
+    KafkaConnectUiWorkerSecurityGroup: # created a secourity group for kafka connect ui
+      Type: AWS::EC2::SecurityGroup
+      Properties:
+        GroupDescription: Security Group for the Fargate Connect Ui Workers.
```

Comment:
Same with my other comment, this one triggered the security hub issue "Unused EC2 security groups should be removed" as well. I'm wondering if it initially starts off not being used unless its manually triggered via the run command on the cli